Project/Source/Components/UI/ComponentCanvasRenderer.h

```diff
@@ -11,7 +11,7 @@ class ComponentCanvasRenderer : public Component {
 	void Save(JsonValue jComponent) const override;
 	void Load(JsonValue jComponent) override;
 	void DuplicateComponent(GameObject& owner) override;
-
+	bool CanBeRemoved() const override;
```

Comment:
Explicitar qué hace que pueda ser eliminado. Returns true when the owner doesn't have X, Y, Z

Reply:
Si no hace nada extra no hace falta. Ya esta explicado en la clase de donde hereda el metodo

Reply:
Pero aunque Component tenga una explicación genérica, creo que sería adecuado explicar cuándo el componente concreto puede ser eliminado, más que nada para no acceder al código

Reply:
Agreed